src/lib.rs

```diff
@@ -60,6 +63,39 @@ pub fn inspect(config: &Config) -> Result<(), InspectError> {
 
     if config.plain {
         println!("{}", output);
+    }
+    else if config.unpretty.starts_with("flowgraph"){
+        if let Some(path) = &config.input {
+            let input_path = PathBuf::from(path);
+            let mut output_path = PathBuf::new();
+            output_path.push(input_path.file_name().unwrap());
```

Comment:
Could the `unwrap()` be a `?`?

Reply:
This unwraps an option. I could change this into something like a `ok_or` and report a specific error if it fails?